backend/migrations/versions/9dcc2f2b8858_add_organization_group_permissions.py

```diff
@@ -0,0 +1,46 @@
+"""add_organization_group_permissions
+
+Revision ID: 9dcc2f2b8858
+Revises: 194608b1ff7f
+Create Date: 2024-04-11 10:34:06.827154
+
+"""
+
+from alembic import op
+from sqlalchemy import orm
+from dataall.core.organizations.db.organization_models import OrganizationGroup, Organization
+from dataall.core.permissions.services.resource_policy_service import ResourcePolicyService
+from dataall.core.permissions.services.organization_permissions import (
+    INVITE_ORGANIZATION_GROUP,
+    REMOVE_ORGANIZATION_GROUP,
+)
+
+# revision identifiers, used by Alembic.
+revision = '9dcc2f2b8858'
+down_revision = '194608b1ff7f'
+branch_labels = None
+depends_on = None
+
+
+def get_session():
+    bind = op.get_bind()
+    session = orm.Session(bind=bind)
+    return session
+
+
```

Comment:
I am testing it at the moment but the issue might be that the permission does not exist yet, we cannot attach `INVITE_ORGANIZATION_GROUP, REMOVE_ORGANIZATION_GROUP` because they do not exist in RDS. Let me check

Reply:
This would be solved by adding a call to `PermissionService.init_permissions(session)`

Reply:
I just tested it and it fails with the error `An error occurred (ResourceNotFound):
            Resource Permission with URI INVITE_ORGANIZATION_GROUP not found` --->
Here is a version that I have tested and it does the trick :) 
```
def upgrade():
    session = get_session()
    print(f"Adding organization group permissions...")
    invite_permission = PermissionService.save_permission(
        session=session,
        name=INVITE_ORGANIZATION_GROUP,
        description='INVITE_ORGANIZATION_GROUP',
        permission_type=PermissionType.RESOURCE.name,
    )
    remove_permission = PermissionService.save_permission(
        session=session,
        name=REMOVE_ORGANIZATION_GROUP,
        description='REMOVE_ORGANIZATION_GROUP',
        permission_type=PermissionType.RESOURCE.name,
    )
    all_org_groups = session.query(OrganizationGroup).all()
    for org_group in all_org_groups:
        ResourcePolicyService.attach_resource_policy(
            session=session,
            group=org_group.groupUri,
            resource_uri=org_group.organizationUri,
            permissions=[INVITE_ORGANIZATION_GROUP, REMOVE_ORGANIZATION_GROUP],
            resource_type=Organization.__name__,
        )
```

Reply:
Ahhha, that's exactly the type of error I was concerned about with 'unclean' deployment)